jdaviz/configs/mosviz/helper.py

```diff
@@ -419,6 +419,14 @@ def load_data(self, spectra_1d=None, spectra_2d=None, images=None,
             String representing the label for the data item loaded via
             ``images``. Can be a list of strings representing data labels
             for each item in ``images`` if  ``images`` is a list.
+
+        directory: str, optional
+            Instead of loading lists of spectra and images, the path to a directory
+            containing all files for a single JWST observation may be given.
+
+        instrument: str, optional
```

Comment:
Might as well use the actual case from code?
```suggestion
        instrument : {'niriss', 'nircam', 'nirspec'}, optional
```